.github/workflows/test.yml

```diff
@@ -7,21 +7,23 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
-
-    runs-on: ubuntu-latest
+  tests:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 2
+      matrix:
+        platform: [ubuntu-latest]
+        python-version: [3.7, 3.8]
```

Comment:
@hirosassa Why not adding 3.9 ? :)

Reply:
@Hi-king thanks, added!

Reply:
@Hi-king `xml.etree.ElementTree.getchildren` is deleted on python 3.9 and test fails.
IMO, skip to support python 3.9 in this PR, and will tackle on next.

https://github.com/m3dev/gokart/runs/1788285866?check_suite_focus=true

Reply:
Oh...This really is problem 😿